src/js/share.js

```diff
@@ -352,33 +369,66 @@
 
 		return shareUrlPromises[maxShares];
 	}
+	
+	function getUserUUID(){
```

Comment:
No need for this function, share-code-web will handle this via the user's session